src/components/Nav.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import hamburgerOpenMenu from '../images/hamburger-open-menu.svg'
+import brigadeLogo from '../images/cfc_logo_2021.svg'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
+
+const Nav = () => {
+  return (
+    <div>
+      <nav class="navbar" aria-label="main">
+        <button aria-label="open" onClick="toggleMenu()" class="menu-icon">
+          <img src={hamburgerOpenMenu} alt="open menu" aria-hidden="true" />
+        </button>
+        <ul class="internal-links">
+          <li id="logo-container">
+            <Link to="/">
+              <img
+                class="brigade-logo"
+                src={brigadeLogo}
+                alt="Code For Chicago Logo"
+              />
+            </Link>
+          </li>
+          <li aria-current="page" class="hideable-int-link">
+            <Link to="/projects">Projects</Link>
+          </li>
+          <li aria-current="page" class="hideable-int-link">
+            <Link to="/portfolio">Portfolio</Link>
+          </li>
+          <li aria-current="page" class="hideable-int-link">
+            <Link to="/join">Join Us</Link>
+          </li>
+          <li aria-current="page" class="hideable-int-link">
+            <Link to="/about">About</Link>
+          </li>
+        </ul>
+        <div class="donate-and-socials">
```

Comment:
Let's remove the unneeded div for now. Just the basic markup will be fine at this point.